sympy/stats/rv.py

```diff
@@ -340,6 +340,55 @@ def __new__(cls, symbol, n, m, pspace=None):
     symbol = property(lambda self: self.args[0])
     pspace = property(lambda self: self.args[3])
 
+class iid(Basic):
```

Comment:
Always try to use full names.

Reply:
Ok, should I change it to `IndependentIdenticallyDistributed`?

Reply:
what about `IndependentIdenticallyDistributedIndexedRandomSymbol`? Or some name that tells you immediately that the variable is a symbol? i.i.d. as a name is more proper for the distribution, not that much for a random symbol.

Reply:
I think the symbol returned can be named something like `IIDRandomIndexedSymbol`, but the class as a whole is a collection of these variables, so it would not be wrong naming it like `IID` or `IndependentIdenticallyDistributed`